sphinx-docs/usermanual/userman_learner.rst

```diff
@@ -0,0 +1,49 @@
+Learner User Manual
+=======================
+
+.. toctree::
+   :maxdepth: 1
+
+ 	userman_learner
```

Comment:
Indentation is important (could be an error?), but also it's redundant I think to include the same file in its own toctree.